### PR TITLE
Task lwdev 9011 remove cashout logic

### PR DESCRIPTION
**Pull request by mmusin**

### CODE REVIEW

File: src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs [hunk 1]
```diff
@@ -193,15 +191,7 @@ protected override void Load(ContainerBuilder builder)
                     .ListeningCommands(typeof(CreateOffchainCashoutRequestCommand), typeof(CreateOffchainCashinRequestCommand))
                         .On(defaultRoute)
                     .WithCommandsHandler<OffchainCommandHandler>(),
-
-                Register.BoundedContext(BoundedContexts.Solarcoin)
-                    .FailedCommandRetryDelay(defaultRetryDelay)
-                    .ListeningCommands(typeof(SolarCashOutCommand))
-                        .On(defaultRoute)
-                    .PublishingEvents(typeof(SolarCashOutCompletedEvent))
```
Comment (polonzer) [inline src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs]: SolarCashOutCompletedEvent class is not deleted. Is it ok?

File: src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs [hunk 2]
```diff
@@ -247,18 +237,6 @@ protected override void Load(ContainerBuilder builder)
...
-                Register.Saga<CashInOutSaga>($"{BoundedContexts.TxHandler}.cash-out-saga")
-                    .ListeningEvents(typeof(CashoutTransactionStateSavedEvent))
-                        .From(BoundedContexts.Operations).On(defaultRoute)
-                    .PublishingCommands(typeof(BitcoinCashOutCommand))
-                        .To(BoundedContexts.Bitcoin).With(defaultPipeline)
-                    .PublishingCommands(typeof(ProcessEthereumCashoutCommand))
-                        .To(BoundedContexts.Ethereum).With(defaultPipeline)
-                    .PublishingCommands(typeof(SolarCashOutCommand))
-                        .To(BoundedContexts.Solarcoin).With(defaultPipeline)
-                    .PublishingCommands(typeof(BlockchainCashoutProcessor.Contract.Commands.StartCashoutCommand))
```
Comment (polonzer) [inline src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs]: StartCashoutCommand class is not deleted. Is it ok?

Comment (mmusin) [reply src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs]: What do you mean?

File: src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs (hunk 1 above)
Comment (mmusin) [reply src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs]: done

File: src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs (hunk 2 above)
Comment (polonzer) [reply src/Lykke.Job.TransactionHandler/Modules/CqrsModule.cs]: Sorry, I thought it is our class.

**Pull request merged**